modules/cnf-working-with-fast-event-notifications-api.adoc

```diff
@@ -0,0 +1,283 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-working-with-fast-event-notifications-api_{context}"]
+= Working with the PTP fast event notifications REST API
+
+You can use the PTP fast event notifications REST API to subscribe an application to the PTP events that are generated for the parent node. PTP fast events notifications are available on each node where a PTP capable network interface is configured.
+
+Events are available from the REST API at `/cluster/node/<node_name>/ptp`, where `<node_name>` is the cluster node running the DU application.
+
+Status requests are sent to `/cluster/node/<node_name>/ptp/status`. Status requests return the PTP event status at the same address.
```

Comment:
Status request are sent by making rest api PUT call to `/subscriptions/status/{subscriptionid}`  returns to return url specified in the subscription for that subscription id.
//pingForSubscribedEventStatus pings for event status  if the publisher  has capability to push event on demand
	// summary: Get status of publishing events.
	// description: If publisher status ping is success, call  will be returned with status accepted.
	// parameters:
	// - name: subscriptionid
	//   description: subscription id to check status for
	// responses:
	//   "201":
	//     "$ref": "#/responses/pubSubResp"
	//   "400":
	//     "$ref": "#/responses/badReq"